BambeeHR/pollen
# AddIcon - PRIVA-1290 - Add Request Icon for Navigation

## Issue(s)
PRIVA-1290
https://bambee.atlassian.net/jira/software/c/projects/PRIVA/boards/88?modal=detail&selectedIssue=PRIVA-1290&assignee=610ad23b4e8d8d00699456e4

https://www.figma.com/file/wKCE9eAmpg7YaWLAvFlzKD/App-%7C-HR-Requests-%7C-2022-MAR

## Description
This is the new icon for Requests in APP Navigation.

## Screenshot(s)
In Storybook (BaseIcon):
![Screenshot_20220616_040628](https://user-images.githubusercontent.com/10406785/174059324-09dca997-9bb9-4404-ae37-e1a4c51dbb1d.png)

In Storybook (HighlightedIcon):
![Screenshot_20220616_040751](https://user-images.githubusercontent.com/10406785/174059377-d6013a90-1a59-4b1c-b84e-a60ae4d8b8bc.png)

From Figma Design:
![Screenshot_20220616_040707](https://user-images.githubusercontent.com/10406785/174059403-d671d5b3-901c-4d55-a7f6-49a1b9ae4a9f.png)

## Dependencies

- Other backend service dependencies
- 3rd party services
- Specific environment variables that need to be setup

## Testing
1. npm run storybook
2. navigate to BaseIcon and see new `request` icon
3. navigate to HighlightedIcon and see new 'request` icon

## Risk Assessment
LOW
